tests/layer_tests/pytorch_tests/test_pad.py

```diff
@@ -244,3 +244,120 @@ def test_reflection_padnd(self, pads, dtype, ie_device, precision, ir_version):
         print(ndim)
         self._test(*self.create_model(pads), ie_device, precision, ir_version,
                    kwargs_to_prepare_input={"ndim": ndim, "dtype": dtype})
+
+class TestReplicatePad1D(PytorchLayerTest):
```

Comment:
Could you make a single class to test 1d, 2d and 3d versions? It is better to not create extra classes

Reply:
sure I would however @rkazants earlier requested to have seperate test suite 
> let us have separate test suite for each 1d, 2d, and 3d cases.

Reply:
@11happy Ok, to not redo it again, leave it as is, but do not make it separate for reflection_pad